### ISSUE TITLE
org.sw.demo.apache.thrift: Build failure

### ISSUE THREAD

**Issue by pschlan** [SoftwareNetwork/sw#86]
`org.sw.demo.apache.thrift` doesn't build since it seems the dependency on boost.locale is not declared.

```
Exception in file D:/dev/cppan2/client2/src/sw/builder/command.cpp:764, function execute1: When executing: [org.sw.demo.apache.thrift-0.13.0]/src/thrift/protocol/TJSONProtocol.cpp
C:/Users/patrick.NET79/.sw/storage/pkg/db/47/225d/src/sdir/lib/cpp/src/thrift/protocol/TJSONProtocol.cpp(22): fatal error C1083: Datei (Include) kann nicht geffnet werden: "boost/locale.hpp": No such file or directory
command failed: exit code = 2
Total errors: 1
```

**Comment by egorpugin** [comment]
Hi,

I have uploaded fixed 0.15.0 thrift version.

**Comment by pschlan** [comment]
Thanks - builds fine now!
